README.md

```diff
@@ -11,7 +11,9 @@
 For more information about release and changelogs please see [Changelog](CHANGELOG.md) or refer to the [Github Releases](https://github.com/bazelbuild/rules_kotlin/releases) page.
 
 # `rules_kotlin` Bazel Compatibility
-- HEAD: `5`, `6`
+- HEAD: `6`, `7`
```

Comment:
I believe HEAD still supports 5.x, i'd need to confirm though.

Reply:
Looks like the rules are using `Runfiles` functions that aren't available on 5.x. https://buildkite.com/bazel/rules-kotlin-kotlin/builds/4120#018e7eb3-adf8-4cec-a020-3f3dec75d98e/122-399